Development.md

```diff
@@ -1,38 +1,57 @@
-# Compile Agent 2 using Docker
+# Build and distribute Stackstate Agent in linux using Docker
 
-Run the stackstate agent runner gitlab docker container:
+Using our builder image clone and checkout the public repo: 
 ```bash
-docker run -ti docker.io/stackstate/stackstate-agent-runner-gitlab:deb_20190429
+$ docker run --rm -ti docker.io/stackstate/stackstate-agent-runner-gitlab:latest7 bash
+
+$ export CI_PROJECT_DIR=/go/src/github.com/StackVista/stackstate-agent && \
+  mkdir -p /go/src/github.com/StackVista && \
+  cd src/github.com/StackVista && \
+  git clone https://github.com/StackVista/stackstate-agent && \
+  cd stackstate-agent && \
+  git checkout upstream-updates-7-21
```

Comment:
nice job updating the readme, not sure whether hard-coding the branch here will throw peopole off here after we merge to master

Reply:
yep will switch to master when this is all done